packages/checkout-sdk/README.md

```diff
@@ -0,0 +1,172 @@
+# Typescript SDK for Composable Commerce Checkout APIs.
+
+## Usage examples
+
+### Browser environment
+
+```html
+<script src="https://unpkg.com/@commercetools/ts-client@latest/dist/commercetools-sdk-client-v3.umd.js"></script>
+<script src="https://unpkg.com/@commercetools/checkout-sdk@latest/dist/commercetools-checkout-sdk.umd.js"></script>
+```
+
+```html
+<script>
+  // global: @commercetools/ts-client
```

Comment:
why is this part commented out?

Reply:
It's not that significant, it just a line comment only show we have a property called `@commercetools/ts-client` on the global `this` or `window` object that can be accessed to create a `client`. It's basically just a description.